packages/larva-patterns/components/c-figcaption/c-figcaption.twig

```diff
@@ -0,0 +1,8 @@
+<figcaption class="c-figcaption {{ c_figcaption_classes }}">
```

Comment:
`figcaption` tag will always be there. do you think that's good idea?

Reply:
Yes, this is in a pattern called c-figcaption so it should only be used within a figure element - it's a way we can enforce semantic HTML with the patterns.